README.md

```diff
@@ -1,4 +1,7 @@
 <p align="center">
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![All Contributors](https://img.shields.io/badge/all_contributors-7-orange.svg?style=flat-square)](#contributors-)
```

Comment:
https://github.com/logdna/logger-node/tree/darinspivey/issue48#readme

Is this being rendered correctly in the README markdown? I'm seeing the literal text here but might be something with my system/browser. Fwiw the README in the all contributors github repo doesn't have the `BADGE` comments

Reply:
Yes, this is rendering correctly AFAICT.  The badges link to the specific change/commit/contribution.  It's all generated, so we have no control over it anyway.

Reply:
![image](https://user-images.githubusercontent.com/28550015/114461878-27800380-9b97-11eb-98b8-e73238dc0a3e.png)

Ah weird, not sure why but this is what I'm seeing on my browser. Otherwise lgtm though so I'll approve

Reply:
nm, I see the issue now.  I'll see if I can fix it manually.

Reply:
FIXED!